funcx_endpoint/funcx_endpoint/cli.py

```diff
@@ -43,21 +44,31 @@ def get_cli_endpoint() -> Endpoint:
     return endpoint
 
 
-def common_options(f):
-    def debug_callback(ctx, param, value):
-        if not value or ctx.resilient_parsing:
-            return
+def log_flag_callback(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+
+    state = CommandState.ensure()
+    setattr(state, param.name, value)
+    setup_logging(debug=state.debug)
 
-        state = CommandState.ensure()
-        state.debug = True
 
+def common_options(f):
     f = click.option(
         "--debug",
         is_flag=True,
-        hidden=True,
         expose_value=False,
-        callback=debug_callback,
+        callback=log_flag_callback,
         is_eager=True,
+        help="Emit extra information generally only helpful for developers",
```

Comment:
Instead of calling out 'developers', how about just describe the usage. ie. 
"Emit extra logging information to console, helpful for developers and others in easier visibility in endpoint activities"

Reply:
Och; GitHub didn't show me this comment until after I had hit the merge button.  That's ... a good suggestion.